formidable/serializers/forms.py

```diff
@@ -77,15 +77,16 @@ def validate(self, data):
     def create(self, validated_data):
         conditions = validated_data.pop('conditions', None)
         instance = super(FormidableSerializer, self).create(validated_data)
-        instance.conditions = conditions
+        if conditions:
+            instance.conditions = conditions
+            instance.save()
         return instance
 
     def update(self, instance, validated_data):
-        conditions = validated_data.pop('conditions', None)
+        instance.conditions = validated_data.pop('conditions', None)
```

Comment:
we can't keep `conditions` inside validated_data because DRF will raise an Exception (about nested values).
If we set `conditions` at the end, we need to call `.save()` again. So I set `conditions` on the object first and super.parent() will do the `save()`